apis/overview/lifecycle.md

```diff
@@ -27,33 +27,62 @@ The following changes are backward compatible:
 - Adding new optional request parameters
 - Adding new fields to the response
 
+As much as possible, we will notify you of a breaking change well before its release. When we release a new version of an API, we provide one year for you to update your applications. The exact amount of notice we provide may vary depending on why the breaking change is necessary.
+
 ## API release status
 
-API product releases have a well-defined lifecycle. The release status determines the level of support as described below.
+API product releases have a well-defined lifecycle. The release status determines the level of support we offer for the API.
+
+![api-lifecycle-diagram](/images/API-Lifecycle-complete.jpg)
+
+### Tech Preview
+
+We may release new APIs, features, and operations as a Tech Preview. This allows you to review upcoming capabilities and provide feedback. Additionally, it assists our team in making sure we are providing the right solutions and expected functionality.
+
+You may use APIs in Tech Preview for free while they are in preview. Do not use these APIs in Production. They are available for testing only, to gather feedback before releasing for General Availability.
```

Comment:
Is this applicable for tech-preview versions of APIs that already have GA version?

Reply:
Not sure. Will have to check

Reply:
Yes, this is applicable for tech-preview versions of APIs with a GA version. The tech-preview version - which will be free - is only meant to be used for testing. Not in production.